database/db.py

```diff
@@ -249,7 +249,7 @@ def updateDatabaseProducts(self, products):
                     self.insertRetailer(x.source)
                     retailer = self.index("SELECT * FROM Retailers WHERE name='" + str(x.source) + "';")
                 retID = retailer[0][0]
-                query = "REPLACE INTO Retailers_Products (product_id, retailer_id, name, price, photo, source, instock, new, price_check) VALUES (" + str(prodID) + "," + str(retID) + ",'" + str(x.name) + "','"+ str(x.price) +"','"+str(x.photo)+"','"+str(x.source)+"','"+str(x.instock)+"','"+str(x.new)+"','"+str(x.price_check)+"');"
+                query = "REPLACE INTO Retailers_Products (product_id, retailer_id, name, price, photo, source, instock, new, price_check, type) VALUES (" + str(prodID) + "," + str(retID) + ",'" + str(x.name) + "','"+ str(x.price) +"','"+str(x.photo)+"','"+str(x.source)+"','"+str(x.instock)+"','"+str(x.new)+"','"+str(x.price_check)+"','" + str(x.product_type) + "');"
```

Comment:
NB (non-blocking comment, i.e. you don't need to change this, but you can if you want. i will approve and merge without this change):

There is an easier way to do this in Python. I should have caught it the first time you submitted DB code but I didn't. That's on me. 
f-strings (format strings) are a handy way to add values into a string. You add them in brackets and Python evaluates them before forcing the evaluated code inside of the bracket to a string. 
https://www.geeksforgeeks.org/formatted-string-literals-f-strings-python/

so if i had:
f"Two + two is {2 + 2}" -> "Two + two is 4" is what would be evaluated

Anyway, you can use this to do things like this: 
"REPLACE INTO Retailers_Products (product_id, retailer_id, name, price, photo, source, instock, new, price_check, type) VALUES ({prodID}, {retID}, '{x.name}', '{x.price}', '{x.photo}', '{x.source}', '{x.instock}', '{x.new}', '{x.price_check}', '{x.product_type}');"